File: src/algorithms/capitalize-title.js
/**
 * Given a string title consisting of one or more words separated by a single
 * space, where each word consists of English letters, capitalizes the string
 * by changing the capitalization of each word such that:
 *     • if the length of the word is 1 or 2 letters, changes all the letters
 *       to lowercase.
 *     • otherwise, changes the first letter to uppercase and the remaining
 *       letters to lowercase.
 * Returns the capitalized parameter title.
 * 
 * @param {string} title - A string of English letters
 * @returns {string} - Returns a modified string
 */
function capitalizeTitle (title) {
  title = title.split(' ');
  for (let i = 0; i < title.length; i++) {
    if (title[i].length > 2) {
      title[i] = (
        title[i].charAt(0).toUpperCase() + title[i].slice(1).toLowerCase()
      );
    } else {
      title[i] = title[i].toLowerCase();
    }
  }
  title = title.join(' ');
  return title;
}

File: src/algorithms/capitalize-title.py
def capitalize_title(title):
    """You are given a string title consisting of one or more words separated 
    by a single space, where each word consists of English letters. Capitalize 
    the string by changing the capitalization of each word such that:
    • If the length of the word is 1 or 2 letters, change all letters to 
    lowercase.
    • Otherwise, change the first letter to uppercase and the remaining letters 
    to lowercase.
    Return the capitalized title.

    Args:
        title (str): A string of English letters.

    Returns:
        str: Returns a modified string.
    """
    title = title.lower()
    title = title.split()
    for i in range(len(title)):
        if len(title[i]) > 2:
            title[i] = title[i].capitalize()
    return " ".join(title)
